platform/src/main/java/org/stellar/anchor/platform/callback/RestUniqueAddressIntegration.java

```diff
@@ -0,0 +1,56 @@
+package org.stellar.anchor.platform.callback;
+
+import static org.stellar.anchor.platform.callback.PlatformIntegrationHelper.*;
+
+import com.google.gson.Gson;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.springframework.http.HttpStatus;
+import org.stellar.anchor.api.callback.GetUniqueAddressResponse;
+import org.stellar.anchor.api.callback.UniqueAddressIntegration;
+import org.stellar.anchor.api.exception.AnchorException;
+import org.stellar.anchor.auth.AuthHelper;
+
+public class RestUniqueAddressIntegration implements UniqueAddressIntegration {
+  private final String anchorEndpoint;
+  private final OkHttpClient httpClient;
+  private final AuthHelper authHelper;
+  private final Gson gson;
+
+  public RestUniqueAddressIntegration(
+      String anchorEndpoint, OkHttpClient httpClient, AuthHelper authHelper, Gson gson) {
+    try {
+      new URI(anchorEndpoint);
+    } catch (URISyntaxException e) {
+      throw new IllegalArgumentException("invalid 'baseUri'");
+    }
+
+    this.anchorEndpoint = anchorEndpoint;
+    this.httpClient = httpClient;
+    this.authHelper = authHelper;
+    this.gson = gson;
+  }
+
+  @Override
+  public GetUniqueAddressResponse getUniqueAddress(String transactionId) throws AnchorException {
+    HttpUrl url =
+        okhttp3.HttpUrl.get(anchorEndpoint)
```

Comment:
You're already importing `import okhttp3.HttpUrl` so you don't need the prefix here:

```suggestion
        HttpUrl.get(anchorEndpoint)
```

Reply:
Improved.